mindmeld/components/entity_recognizer.py

```diff
@@ -258,7 +274,9 @@ def predict_proba(
             query = self._resource_loader.query_factory.create_query(
                 query, time_zone=time_zone, timestamp=timestamp
             )
-        predict_proba_result = self._model.predict_proba([query])
+        predict_proba_result = self._model.predict_proba(
```

Comment:
Again, we should not be passing in the system entity recognizer everytime. We should be modeling the call patterns of the system entity recognizer similar to the resource loader.

Reply:
done